docs/artifacts/where-the-plugin-lives.html

```
@@ -0,0 +1,156 @@
+<title>Where the Plugin Lives</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)